docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
+| [Number](#3-number)             | Detects number from the text.                                | 50 rs per person, ५ किलो चावल, मुझे एक लीटर ऑइल चाहिए          |
+| [Phone number](#4-phone-number) | Detects phone numbers in given text.                         | +919222222222                                                |
+| [Email](#5-email)               | Detects email addresses in given text.                       | amans.rlx@gmail.com                                          |
+| [Text](#6-text)                 | Detect custom entities in text string using full text search in Datastore or based on contextual model | **pizza**, **मुंबई**                                           |
+| [PNR-Number](#7-pnr-number)     | Detect PNR (serial) codes in given text.                     | My flight PNR is **4SGX3E**                                  |
+| [Regex](#8-regex)               | Detect entities using custom regex patterns                  | My flight PNR is **4SGX3E**<br />Please apply **CASH20** coupon code |
 
-Following are the list of different entity types along with its API call:
 
-### text
 
-- This functionality calls the TextDetection class to detect textual entities.
+### APIs Parameters
 
-- Example:
+Following are the list of parameters accepted by APIs used for detection:
 
-  - Example 1: 
+1. **message**: Message on which detection logic needs to run. It is an unstructured text from which entity needs to be extracted. For example, *"I want to order pizza"*.
 
-    - ```python
-      message='i want to order chinese from  mainland china and pizza from domminos'
-      entity_name='restaurant'
-      structured_value=None
-      fallback_value=None
-      bot_message=None
-      ```
+2. **entity_name**: Name of the entity. This parameter primarily defines the key of the output dictionary. 
 
-    - *Python:* 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_text
-      output = get_text(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-      The above can also be done from within the Docker container's shell. Setup is in docker.md file.
+   For example, in Text detection logic  the entity name is based on dictionary name of entities we need to detect.
 
-    - *CURL command:*
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
+   - For detecting:
+     - *cuisine* entity_name will be *"cuisine"*
+     - *dish* entity_name will be *"dish"*
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20want%20to%20order%20chinese%20from%20%20mainland%20china%20and%20pizza%20from%20domminos&entity_name=restaurant&structured_value=&fallback_value=None&bot_message='
-      ```
+   In other detection like `date` or `tim` `entity_name` does not have any siginificance as detection logic works solely based on their entity_types only.
```

Comment:
Spell mistake
tim -> time
siginificance -> significance

Reply:
`entity_name` needs a more clear explanation, saying `does not have any siginificance` implies it must be optional but it is not.

Rather we should explain that often in a chat flow you need multiple entities of the same type, so you would want to name each entity uniquely. This also acts as mask internally to avoid overlapping detections and special detection logic can be customized based on value of this argument

Reply:
@chiragjn Please help me with content, I am not able to explain clearly with the text as you mentioned.